packages/react/src/components/DataGrid/index.tsx

```diff
@@ -30,12 +30,19 @@ export interface ColumnField {
 }
 
 export interface ColumnCustomCell<T extends Row = Row> {
+  id?: string;
+  field: string;
+  renderCell: (params: { row: T }) => React.ReactNode;
+}
+export interface ColumnDisplayCell<T extends Row = Row> {
```

Comment:
very nit-picking: I feel this naming could be improved, reading the code, it's easy to guess what are the responsibilities of `ColumnField` and `ColumnCustomCell`, their naming are perfect. `ColumnDisplayCell` is a bit less explicit. It's a personal feeling, please feel **absolutely** free to ignore this nit-picking comment.

Reply:
Ok I get your point, tbh I used the naming from React-Table ( https://tanstack.com/table/latest/docs/guide/column-defs#column-def-types )